src/pydata_sphinx_theme/theme/pydata_sphinx_theme/theme.conf

```diff
@@ -33,7 +33,7 @@ navbar_center = navbar-nav.html
 navbar_end = theme-switcher.html, navbar-icon-links.html
 left_sidebar_end = sidebar-ethical-ads.html
 footer_items = copyright.html, sphinx-version.html
-page_sidebar_items = page-toc.html, edit-this-page.html
+page_sidebar_items = page-toc.html, edit-this-page.html, sourcelink.html
```

Comment:
I think I'm +1 on adding this by default, though we should probably give a little bit of time to make sure others do not object